Docker/Dockerfile

```diff
@@ -58,6 +58,8 @@ ENV DATA_PATH ''
 ENV FRESHRSS_ENV ''
 ENV LISTEN ''
 ENV OIDC_ENABLED ''
+ENV OIDC_REMOTE_USER_CLAIM 'preferred_username'
+ENV OIDC_SCOPES 'openid'
```

Comment:
If possible, I would rather put the default values in `FreshRSS.Apache.conf` instead of in each of our Dockerfiles